PSReadLine/YankPaste.vi.cs

```diff
@@ -247,10 +247,13 @@ public static void ViYankEndOfGlob(ConsoleKeyInfo? key = null, object arg = null
         /// </summary>
         public static void ViYankBeginningOfLine(ConsoleKeyInfo? key = null, object arg = null)
         {
-            int length = _singleton._current;
+            var start = GetBeginningOfLinePos(_singleton._current);
+            var length = _singleton._current - start; 
             if (length > 0)
             {
-                _singleton.SaveToClipboard(0, length);
+                _clipboard.Record(_singleton._buffer, start, length);
+                _singleton._current = start;
+                _singleton.Render();
```

Comment:
Why do you choose to call `Render` instead of just moving the cursor?

Reply:
I'll have a better look at the `MoveCursor` method.

Reply:
I'm about to publish `2.0.0-beta6` before Friday this week. I want to include this fix if time permits, but no rush, it can always catch the next release.

Reply:
Done.

Reply:
Thanks @springcomp!